tests/suites/cloud/test-app/containerA/Dockerfile.template

```diff
@@ -0,0 +1,3 @@
+FROM balenalib/%%BALENA_ARCH%%-alpine
```

Comment:
NIT: We should be using a tagged image for base images even with the use case we got.